specification/communication/data-plane/PhoneNumbers/readme.md.bak

```diff
@@ -0,0 +1,77 @@
+# communicationservices
```

Comment:
Looks like this was committed accidentally?

Reply:
Yes, I'll delete this file